config/abilities/provider_admin.rb

```diff
@@ -76,6 +76,10 @@
       service.account_id == user.account_id && can?(:manage, :multiple_services) and not service.default_or_last?
     end
 
+    if account.provider_can_use?(:api_as_product)
+      can :edit, BackendApi
```

Comment:
Probably `can :manage, BackendApi`

Reply:
I'll leave this change for when we start the permissions pert of APIAP.